common/decklist.cpp

```diff
@@ -466,66 +466,111 @@ bool DeckList::saveToFile_Native(QIODevice *device)
 bool DeckList::loadFromStream_Plain(QTextStream &in)
```

Comment:
Do you have a timescale on when? Its better to "eat the frog" now than procrastinate on it. Its too easy to get a PR merged and either not come back to it, or leave more work for someone else.